hs/AboutResponse.hs

```diff
@@ -5,9 +5,11 @@ module AboutResponse where
 import           Text.Blaze ((!))
 import qualified Text.Blaze.Html5 as H
 import qualified Text.Blaze.Html5.Attributes as A
+import           Text.Blaze.Html.Renderer.Text (renderHtml)
+import Data.Text.Lazy
```

Comment:
For this and the one below it, please indicate which function is being imported, since I think there's only one.